roles/press/templates/etc/supervisor/conf.d/press.conf

```diff
@@ -4,4 +4,4 @@ directory=/var/cnx/apps/press
 numprocs={{ hostvars[inventory_hostname].press_count|default(1) }}
 process_name=%(program_name)s-%(process_num)s
 environment=SHARED_DIR="/var/cnx/apps/press/var",DB_URL="postgresql://{{ press_db_user }}:{{ press_db_password }}@{{ press_db_host }}:{{ press_db_port }}/{{ press_db_name }}",DB_SUPER_URL="postgresql://postgres@{{ press_db_host }}:{{ press_db_port }}/{{ press_db_name }}",SENTRY_DSN="{{ sentry_dsn|default() }}"
-command=/var/cnx/venvs/press/bin/gunicorn -b 0.0.0.0:6543 --access-logfile - --error-logfile - -n press --reload wsgi:app
+command=/var/cnx/venvs/press/bin/gunicorn -b 0.0.0.0:{{ press_base_port|default(default_press_base_port) }} --access-logfile - --error-logfile - -n press --reload wsgi:app
```

Comment:
Are we experimenting w/ a different server here? What advantages does gunicorn give us over waitress? I've seen comments re: gunicorn allowing massive parallel hammering of the DB server (auto-scaling of number of workers) Do we need to control for that? Can't we specify gunicorn in the ini file, and still launch via pserve? (for a consistent devops experience :-)

Reply:
This is outside the scope of this PR, but feel free to create an issue for it.

Reply:
Ah I see that it's already gunicorn. Definitely want to think about that.